### PR TITLE
Convert admin lookup handler to hooks

### PR DESCRIPTION
**Pull request by haykam821**
This pull request removes the admin lookup handler which has code mostly duplicated from the default one, and instead adds in hooks registered and unregistered alongside other admin features. This allows admins to be able to use the hooks feature and gets rid of the need to support two different lookup handlers at once.

### CODE REVIEW

**Comment by FlyingSixtySix** [comment]
I'd prefer if the moderator actions were on a single line, and used text instead of emojis.

**Comment by haykam821** [comment]
Here is what they look like now:

image

**Comment by FlyingSixtySix** [comment]
The button font size looks a bit small on my machine.
image

**Comment by haykam821** [comment]
I did that to distinguish between lookups' regular buttons (Report and Close) and hook buttons. Would you prefer them to be normal-sized, like the below example?

Lookup with admin hooks and normal-sized buttons

**Comment by FlyingSixtySix** [comment]
Yes, please.

**Comment by haykam821** [comment]
It has been done.

File: resources/public/admin/admin.css [hunk 1]
```diff
@@ -2,6 +2,11 @@
 | Admin Box
 \*\\\\\\\\\\\\\\\\\\\\\\\\\\*/
 
+pre {
+    display: inline;
```
Comment (Sorunome) [inline resources/public/admin/admin.css]: Why make a `<pre>` element inline, that is what `<code>` or `<span>` elements are for! `<pre>` should stay a block element

File: resources/public/admin/admin.js [hunk 2]
```diff
@@ -297,62 +295,46 @@
...
-                },
+                /**
+                 * Register hooks for admin-specific lookups.
+                 */
                 init: function () {
-                    admin.lookup.registerHandle(self.create);
+                    App.lookup.registerHook({
+                        id: "login",
+                        name: "Login",
+                        get: data => $("<pre>").text(data.login),
```
Comment (Sorunome) [inline resources/public/admin/admin.js]: same down here with using `<pre>` as inline element

File: resources/public/admin/admin.js [hunk 3]
```diff
@@ -297,62 +295,46 @@
...
+                    }, {
+                        id: "user_agent",
+                        name: "User Agent",
+                        get: data => $("<pre>").text(data.userAgent),
```
Comment (Sorunome) [inline resources/public/admin/admin.js]: abd here

**Pull request merged**